### ISSUE TITLE
Refused to connect

### ISSUE THREAD

**Issue by mf-social** [hassio-addons/addon-grafana#41]
# Problem/Motivation

I'm trying to setup glances/grafana/influxDB as per your video, but I cannot access the grafana webUI and I'm stumped.

## Expected behavior

Being able to access the webUI

## Actual behavior

"This site can’t be reached (URL) refused to connect."

(Note all other webUI's are working no problem - this is the only addon I have that doesn't have an 'add-to-sidebar' but all the others work fine from the 'add-to-sidebar' and from the Open WebUI button)

## Steps to reproduce

Basically, clicking 'open webUI' gives me that error, I've tried using the local IP instead but I get the same error.  If it helps I'm using the Ngnix addon too (no idea if related)

I totally understand that this is going to eb something my end, any ideas?

**Comment by frenck** [comment]
Please supply logs.

**Comment by mf-social** [comment]
Thanks

Grafana add-on logs...

```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 00-banner.sh: executing... 

-----------------------------------------------------------
 Hass.io Add-on: Grafana
 The open platform for beautiful analytics and monitoring
-----------------------------------------------------------
 Add-on version: 2.2.1
 You are running the latest version of this add-on.
 System: Debian GNU/Linux 10 (buster)  (amd64 / qemux86-64)
 Home Assistant version: 0.96.0b1
 Supervisor version: 168
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
-----------------------------------------------------------
[cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing... 
[cont-init.d] 01-log-level.sh: exited 0.
[cont-init.d] grafana.sh: executing... 
[cont-init.d] grafana.sh: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[17:12:25] INFO: Starting Memcached...
[17:12:25] INFO: Starting Grafana...
t=2019-07-12T17:12:26+0100 lvl=info msg="Starting Grafana" logger=server version=6.2.5 commit=6082d19 branch=HEAD compiled=2019-06-25T18:56:19+0100
t=2019-07-12T17:12:26+0100 lvl=info msg="Config loaded from" logger=settings file=/usr/share/grafana/conf/defaults.ini
t=2019-07-12T17:12:26+0100 lvl=info msg="Config loaded from" logger=settings file=/etc/grafana/grafana.ini
t=2019-07-12T17:12:26+0100 lvl=info msg="Path Home" logger=settings path=/usr/share/grafana
t=2019-07-12T17:12:26+0100 lvl=info msg="Path Data" logger=settings path=/data
t=2019-07-12T17:12:26+0100 lvl=info msg="Path Logs" logger=settings path=/var/logs/grafana
t=2019-07-12T17:12:26+0100 lvl=info msg="Path Plugins" logger=settings path=/var/lib/grafana/plugins
t=2019-07-12T17:12:26+0100 lvl=info msg="Path Provisioning" logger=settings path=/usr/share/grafana/conf/provisioning
t=2019-07-12T17:12:26+0100 lvl=info msg="App mode production" logger=settings
t=2019-07-12T17:12:26+0100 lvl=info msg="Initializing SqlStore" logger=server
t=2019-07-12T17:12:26+0100 lvl=info msg="Connecting to DB" logger=sqlstore dbtype=sqlite3
t=2019-07-12T17:12:26+0100 lvl=info msg="Starting DB migration" logger=migrator
t=2019-07-12T17:12:26+0100 lvl=info msg="Initializing HTTPServer" logger=server
t=2019-07-12T17:12:26+0100 lvl=info msg="Initializing InternalMetricsService" logger=server
t=2019-07-12T17:12:26+0100 lvl=info msg="Initializing RemoteCache" logger=server
t=2019-07-12T17:12:26+0100 lvl=info msg="Initializing QuotaService" logger=server
t=2019-07-12T17:12:26+0100 lvl=info msg="Initializing LoginService" logger=server
t=2019-07-12T17:12:26+0100 lvl=info msg="Initializing PluginManager" logger=server
t=2019-07-12T17:12:26+0100 lvl=info msg="Starting plugin search" logger=plugins
t=2019-07-12T17:12:26+0100 lvl=info msg="Plugin dir created" logger=plugins dir=/var/lib/grafana/plugins
t=2019-07-12T17:12:26+0100 lvl=info msg="Initializing RenderingService" logger=server
t=2019-07-12T17:12:26+0100 lvl=info msg="Initializing AlertingService" logger=server
t=2019-07-12T17:12:26+0100 lvl=info msg="Initializing DatasourceCacheService" logger=server
t=2019-07-12T17:12:26+0100 lvl=info msg="Initializing HooksService" logger=server
t=2019-07-12T17:12:26+0100 lvl=info msg="Initializing SearchService" logger=server
t=2019-07-12T17:12:26+0100 lvl=info msg="Initializing ServerLockService" logger=server
t=2019-07-12T17:12:26+0100 lvl=info msg="Initializing TracingService" logger=server
t=2019-07-12T17:12:26+0100 lvl=info msg="Initializing UsageStatsService" logger=server
t=2019-07-12T17:12:26+0100 lvl=info msg="Initializing UserAuthTokenService" logger=server
t=2019-07-12T17:12:26+0100 lvl=info msg="Initializing CleanUpService" logger=server
t=2019-07-12T17:12:26+0100 lvl=info msg="Initializing NotificationService" logger=server
t=2019-07-12T17:12:26+0100 lvl=info msg="Initializing provisioningServiceImpl" logger=server
t=2019-07-12T17:12:26+0100 lvl=info msg="Initializing Stream Manager"
t=2019-07-12T17:12:26+0100 lvl=info msg="HTTP Server Listen" logger=http.server address=0.0.0.0:80 protocol=http subUrl= socket=
```

presumed normal

**Comment by frenck** [comment]
This is the starting sequence, got the end as well?

**Comment by mf-social** [comment]
That's as far as it goes :/

Presume then that it's not normal to stop there 😃

**Comment by mf-social** [comment]
...just uninstalled and reinstalled the addon - stopping at exactly the same place in the logs - add-on shows as running, just stuck there - any ideas?

**Comment by frenck** [comment]
Oh sorry my bad, it is the full log and it is looking great! So the add-on is running.
This means you are having other issues.

**Comment by mf-social** [comment]
Any thoughts on what those issues might be or how to diagnosis please?

**Comment by frenck** [comment]
How are you trying to access it?
Made sure to access it directly on the local network before using anything else.
E.g., `http://IP_ADDRESS_OF_HASSIO:3000`

**Comment by mf-social** [comment]
Well, I rebooted the whole machine and it is now working. So no idea what the issue was, but apparently now solved.

Thanks for your time @frenck :+1:
